### DISCUSSION TITLE
Nginx

### DISCUSSION THREAD

**Discussion by Notadumdum** [miguelgrinberg/python-socketio#1127] [Q&A] [answered]
Goodday I want to make use of nginx reverse proxy to send origin header to a socket.io server and I don't know how I'd connect my code to nginx to my socket.io client

```server {
    listen 443 ssl;
    server_name localhost;
    ssl_certificate /path/to/ssl_certificate.crt;
    ssl_certificate_key /path/to/ssl_private_key.key;

    location /socket.io/{
        proxy_pass wss://socketioexample.com/socket.io/ ;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
        proxy_set_header Host $host;
        proxy_set_header X-Real-IP $remote_addr;
        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
        proxy_set_header origin 'https://example.com';
    }
}
```
I'm aware that i have to input this in the nginx.conf file but after this I have not much of an idea of how to configure the connection when entering vscode...

**Comment by miguelgrinberg** [comment, marked as answer]
You don't need Nginx for this. Send the `Origin` header directly in your connection call.

https://python-socketio.readthedocs.io/en/latest/api.html#socketio.Client.connect
